scripts/build.py

```diff
@@ -130,11 +131,13 @@ def build_files(source, target, ignore):
                 continue
             target_path = source_path.replace(source, target)
 
-            # Only copy files in ignore or with extensions mentioned in
-            # FILE_EXTENSIONS_TO_COPY_WITHOUT_MINIFICATION variable.
-            file_patterns_to_ignore = (
-                ignore + FILE_EXTENSIONS_TO_COPY_WITHOUT_MINIFICATION)
-            if any(p in source_path for p in file_patterns_to_ignore):
+            # Ignore files with certain extensions
+            if any(p in source_path for p in FILE_EXTENSIONS_TO_IGNORE):
+                continue
+            # Copy files in copy_without_minification or with extensions other
+            # than html, css and js directly.
+            if (any(p in source_path for p in copy_without_minification) or
+                    any(p not in source_path for p in ['html', 'css', 'js'])):
```

Comment:
Though, I think it'd be better to get rid of copy_without_minification entirely. It just confuses things. Once that's done, we can say: if the filepath is in ['html', 'css', 'js'], do one thing, otherwise do the other thing.